sycl/source/detail/platform_impl.hpp

```diff
@@ -180,12 +182,16 @@ class platform_impl {
   static std::shared_ptr<platform_impl>
   getPlatformFromPiDevice(RT::PiDevice PiDevice, const plugin &Plugin);
 
+  context getDefaultContext();
+
 private:
   bool MHostPlatform = false;
   RT::PiPlatform MPlatform = 0;
   std::shared_ptr<plugin> MPlugin;
   std::vector<std::weak_ptr<device_impl>> MDeviceCache;
   std::mutex MDeviceMapMutex;
+  std::vector<context> MDefaultContexts;
```

Comment:
Since the doc now says that there is one default context per platform, does it make sense to have just a value here instead of `std::vector`?

Reply:
Yes